File: python/get.py
import http.client as hc

host = "httpbin.org"

method = "GET"

url = "/"

conn = hc.HTTPConnection(host)

conn.request(method, url)

resp = conn.getresponse()

if resp.status != 200:
    print(f"Status: {resp.status}, reason: {resp.reason}")
else:
    body = resp.read().decode("utf-8")
    print(body)

conn.close()

File: go/get.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
)

func main() {
	url := "your_endpoint"

	resp, err := http.Get(url)

	if err != nil {
		fmt.Println(err)
	}

	defer resp.Body.Close()

	body, err := ioutil.ReadAll(resp.Body)

	fmt.Println(string(body))
}
